index.html

```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
+        <h2 class="card-text2">Skills</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc2">Codekit</a></li>
+          <li><a href="#" class="anc2">Github</a></li>
+          <li><a href="#" class="anc2">Codepen</a></li>
+          <li><a href="#" class="anc2">Gitlab</a></li>
+          <li><a href="#" class="anc2">Terminal</a></li>
+        </ul>
+      </div>
+     </div>
+    </section>
 
-    <div class="card">
-      <div class="card-image light">
-        <img src="images/light.svg" class="lt-image" alt="">
+
+    <!-- get in touch -->
+
+    <div class="contact-container">
+      <div class="contact-me">
+      <div class="contact-message">
+        <p class="contact-info">I'm always interested in hearing about new projects, so if you'd like to chat please get
+          in touch.</p>
       </div>
-      <h2 class="card-text2">Skills</h2>
-      <ul class="alist">
-        <li><a href="#" class="anc2">Codekit</a></li>
-        <li><a href="#" class="anc2">Github</a></li>
-        <li><a href="#" class="anc2">Codepen</a></li>
-        <li><a href="#" class="anc2">Gitlab</a></li>
-        <li><a href="#" class="anc2">Terminal</a></li>
-      </ul>
-   </div>
-  </section>
+      <div class="contact-forms">
+        <form class="form" action="https://formspree.io/f/xrgdnqoz" method="post">
+          <ul class="inputs">
+            <li>
+              <label for="name" class="label"></label>
+              <input type="text" name="user-name" value="" id="name" placeholder="Full Name" maxlength="30" class="in"
+                required="required">
+            </li>
 
+            <li>
+              <label for="mail" class="label"></label>
+              <input type="email" name="user-email" value="" id="e-mail" placeholder="E-mail address" class="in"
+                required="required>
+           </li>
 
-   <!-- get in touch -->
-   <div class="contact">
-     <p class="contact-info">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+           <li>
+            <label for=" msg" class="label"></label>
+              <textarea required name="user-message" rows="" cols="" id="msg" placeholder="Enter text here"
+                maxlength="500" class="in"></textarea>
+            </li>
+          </ul>
 
-   </div>
+          <button type="submit" class="btn btn-success see-button">Get In Touch</button>
 
-  </section>
+        </form>
+      </div>
+      </div>
+
+      <div class="end">
+      <ul class="social-icons">
+        <li class="icon"><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/medium.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/github.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/angel.svg" alt=""></a></li>
+      </ul>
+        </div>
```

Comment:
- [ ] You have done a great job so far with your desktop design 👏 👏. But there are some little differences between your design and the design in the Figma template you have chosen. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

- Your project Design👇 (Please pay attention to my highlight below in the contact section of your project):

1.  In the `Desktop` version of this particular template you need to add one more `input` tag to your form that will hold your `Last Name`.
2. Kindly check the `background image` of your design as it is coming under the `form` and the `text`. Kindly check the Figma design and make adjustments.

Your design 👇 

![dan1](https://user-images.githubusercontent.com/78634841/206758328-4d33b858-c2a2-42e1-8c00-7be7b936deb7.png)

Figma Design 👇 

![cont real](https://user-images.githubusercontent.com/78634841/206758097-d320f6a4-6d12-49ff-a9e6-a409121e49d6.png)

Reply:
we were instructed to ignore the additional input field for name and use 1 field on desktop, like the one on mobile